gaphor/core/modeling/coremodel.py

```diff
@@ -54,6 +54,11 @@ class RefChange(PendingChange):
     property_ref: _attribute[str] = _attribute("property_ref", str)
 
 
+class Picture(Element):
+    content: _attribute[str] = _attribute("content", str)
+    dimension: _attribute[str] = _attribute("dimension", str)
```

Comment:
Shouldn't we just use a width and height property?

```suggestion
    width: _attribute[int] = _attribute("width", int)
    height: _attribute[int] = _attribute("height", int)
```

Reply:
Let me try it out whether it works as expected.

Reply:
Commited a full change.

Reply:
@danyeaw can you please run the makefile for image generation, it still crashing for me :(

Reply:
Done!